01-reading.Rmd

```diff
@@ -504,21 +505,63 @@ no_official_lang_data <- filter(can_lang, category != "Official languages")
 write_csv(no_official_lang_data, "data/no_official_languages.csv")
 ```
 
-## Scraping data off the web using R
-
-In the first part of this chapter, we learned how to read in data from plain text files that are usually "rectangular" in shape using the `tidyverse` `read_*` functions. Sadly, not all data comes in this simple format, but we can happily use many other tools to read in more messy/wild data formats. One common place people often want/need to read in data from is websites. Such data exists in a non-rectangular format. One quick and easy solution to get this data is to copy and paste it. However, this becomes painstakingly long and boring when there is a lot of data that needs gathering. And any time you start doing a lot of copying and pasting, you will likely introduce errors. 
-
-The formal name for gathering non-rectangular data from the web and transforming it into a more useful format for data analysis is **web scraping**. There are two different ways to do web scraping: 1) screen scraping (similar to copying and pasting from a website, but done in a programmatic way to minimize errors and maximize efficiency) and 2) web APIs (**a**pplication **p**rogramming **i**nterface) (a website that provides a programatic way of returning the data as JSON or XML files via http requests). In this course, we will explore the first method, screen scraping using R's [`rvest` package](https://github.com/hadley/rvest).
-
-### HTML and CSS selectors
-
-Before we jump into scraping, let's set up some motivation and learn a little bit about what the "source code" of a website looks like. Say we are interested in knowing the average rental price (per square footage) of the most recently available one-bedroom apartments in Vancouver from https://vancouver.craigslist.org. When we visit the Vancouver Craigslist website and search for one-bedroom apartments, this is what we are shown:
+## Obtaining data from the web 
+
+> This section is not required reading for the remainder of the textbook.
```

Comment:
```suggestion
> *This section does not contain pre-requisite knowledge for the remainder of the topics in the textbook, and is included for those leaners interested in learning a little bit more about how to obtain different types of data from the web.*
```

Reply:
i think the original wording is a bit clearer to be honest. i wouldn't mind adding something like your "is included for those...." to explain who should read the section though. Thoughts?

Reply:
OK how about this then:

This section is not required reading for the remainder of the textbook. It is included for those leaners interested in learning a little bit more about how to obtain different types of data from the web.

I want to add the second sentence as the first read a bit terse to me, and it left me wondering if it was not required reading, why is it here.